### PR TITLE
Update license.

### PR DESCRIPTION
**Pull request by totalo**

### CODE REVIEW

**Review by kezhenxu94 — APPROVED** [review]

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -1,18 +1,21 @@
 /*
- * Copyright 2007-present the original author or authors.
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (wu-sheng) [inline .mvn/wrapper/MavenWrapperDownloader.java:2]: Why change back to ASF?

**Review by wu-sheng — CHANGES_REQUESTED** [review]

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (wu-sheng) [reply .mvn/wrapper/MavenWrapperDownloader.java:2]: https://github.com/apache/shardingsphere/blob/c2df0553596ccd24646fafe1ab11b63baa81c8d3/.mvn/wrapper/MavenWrapperDownloader.java#L2-L14

ShardingSphere is using this too.

Comment (wu-sheng) [reply .mvn/wrapper/MavenWrapperDownloader.java:2]: At https://github.com/apache/shardingsphere/pull/11883#discussion_r691303345, you said, the `Licensed to the Apache Software Foundation (ASF) under one or more` comes from SkyWalking, but from this change log, SkyWalking has the correct and original header.

Could you share what is missing here?

Comment (totalo) [reply .mvn/wrapper/MavenWrapperDownloader.java:2]: Sorry, I think I might understand it wrong.

Comment (wu-sheng) [reply .mvn/wrapper/MavenWrapperDownloader.java:2]: OK, get it.

**Pull request closed without merging**